test/Backends/ArithmeticCircuit.hs

```diff
@@ -28,10 +29,34 @@ equalTerm = J.Lam "x" (J.Var "x")
 equalType :: Type
 equalType = J.Pi Omega (J.PrimTy ()) (J.PrimTy ())
 
-equalCircuit :: C.ArithCircuit Fr
-equalCircuit =
-  C.ArithCircuit
-    { C.weights = C.GateWeights {C.wL = [], C.wR = [], C.wO = []},
-      C.commitmentWeights = [],
-      C.cs = []
-    }
+power :: Term
+power = J.LamM "x" (J.LamM "w" (Base.eq (Base.exp (J.Var "w") (Base.c 2)) (J.Var "x")))
+
+quadratic :: Parameterisation.Integer a b => a → b → Term
```

Comment:
Are these tested with proofs somewhere (we should do so)?

Reply:
They're been used with `zkp` now, update to follow.